reactivesocket-core/src/main/java/io/reactivesocket/exceptions/Exceptions.java

```diff
@@ -30,9 +30,15 @@ public static Throwable from(Frame frame) {
         final int errorCode = Frame.Error.errorCode(frame);
         String message = "<empty message>";
         final ByteBuffer byteBuffer = frame.getMetadata();
-        if (byteBuffer.hasArray()) {
```

Comment:
This code is defaulting to `<empty message>` if the buffer is not array backed, which isn't correct but is safe, rite?

Moreover, it is assuming `UTF-8` which isn't correct at all and it is converting entire metadata frame into the message. So, I guess ignoring error message if the buffer isn't array backed, isn't the bigger problem here 😄